apps/ledger-live-desktop/tests/page/modal/add.account.modal.ts

```diff
@@ -25,12 +25,35 @@ export class AddAccountModal extends Modal {
   async selectToken(token: Token) {
     await this.selectAccount.click();
     await this.selectAccountInput.fill(token.tokenName);
-    await this.selectTokenNetwork(token).click();
+    if (await this.selectTokenNetwork(token).isVisible()) {
+      await this.selectTokenNetwork(token).click();
+    } else {
+      const dropdownSize: any = await this.dropdownOptionsList.boundingBox();
+      await this.page.mouse.move(
```

Comment:
what is the purpose of this line if normally it should be enough just to call selectAccountByScrolling() since it has scrolling action inside it?